.github/workflows/run-validation.yml

```diff
@@ -0,0 +1,67 @@
+name: Run validation with RoboValidate
+
+on:
+    # Run on any branch so validate branch can always run.
+    push:
```

Comment:
Short Post is missing any content http://digitalgov.lndo.site/admin/content?title=&type=short_post&status=All
This is a new data structure, but not new content. Grab at least 1 news, resource, and service where 'source_url:' has a value. You'll find those in the content/resource, content/service, and content/news dirs.